[editorial] add `var` info to `let`/`const` section (#2413)
airbnb/javascript: README.md

      count += 1;
    }
    ```
  <a name="references--block-scope"></a><a name="2.3"></a>
  - [2.3](#references--block-scope) Note that both `let` and `const` are block-scoped. Whereas `var` is function scoped.

    ```javascript
    // const and let only exist in the blocks they are defined in.
    {
      let a = 1;
      const b = 1;
      var c = 1;
    }
    console.log(a); // ReferenceError
    console.log(b); // ReferenceError
    console.log(c); // Prints 1
    ```

>     In above code, you can see that the `a` and `b` will generate reference error and `c` prints the number. It is because `a` and `b` are block scoped while `c` is not.

Reviewer: ```suggestion
    In above code, you can see that the `a` and `b` will generate a ReferenceError, and `c` prints the number. This is because `a` and `b` are block scoped, while `c` is scoped to the containing function.
```

    console.log(a); // ReferenceError
    console.log(b); // ReferenceError
    ```

>   <a name="references--difference-let-var"></a><a name="2.4"></a>

Reviewer: ```suggestion
  <a name="references--difference-let-var"></a></a>
```
new sections don't get a numbered link; only legacy ones do.

  - [2.4](#references--difference-let-var) Difference between `let` and `const`
    

>     For loop with **var** keyword

Reviewer 2: In the JS variable var is define in a scope local. we can recall it in a global scope.

Only in JS we can do it.

Variable let is working only in local scope.

Reviewer: I'm not sure what this comment means. `var`, `let`, and `const` are all local scope - var is function scope, and the other two are block scope. None of them are globally scoped.

    ```javascript
    for (var i = 1; i <= 2; i++ ) {
      console.log(i)
    }
    console.log(i) // Prints 3
    ```
    For loop with **let** keyword
    ```javascript
    for (let j = 1; j <= 2; j++ ) {
      console.log(j)
    }
    console.log(j) // Error

>     ```

Reviewer: while explaining the difference is helpful, this guide forbids both `var` and loops, so i'm not sure this is the best example. Perhaps something that shows the hoisting differences - maybe just adding a `var` to the previous section?

Author: I will open a new PR for the same. Thank you for your assistance.

Reviewer: Please do not open a PR; please repurpose this one instead.
